app/src/main/java/org/wikipedia/analytics/eventplatform/ImageRecommendationsEvent.kt

```diff
@@ -0,0 +1,41 @@
+package org.wikipedia.analytics.eventplatform
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import org.wikipedia.WikipediaApp
+
+@Serializable
+@SerialName("/analytics/mobile_apps/android_image_recommendation_event/1.0.0")
+class ImageRecommendationsEvent(
+   private val action: String,
+   private val active_interface: String,
+   private val action_data: String,
+   private val primary_language: String,
+   private val wiki_id: String
+) : MobileAppsEvent(STREAM_NAME) {
+
+    companion object {
+        private const val ACTION_IMPRESSION = "impression"
```

Comment:
Is it necessary to have a `const` for the "impression" string, when we don't have any other constants for any of the other strings?